### PR TITLE
Modorg 43

### PR DESCRIPTION
**Pull request by SinghAdes**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
   <properties>
     <aspectj.version>1.9.7</aspectj.version>
-    <raml-module-builder.version>34.1.0</raml-module-builder.version>
-    <vertx.version>4.3.1</vertx.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version>
```
Comment (grabsefx) [inline pom.xml:18]: please upgrade to the latest 35.0.1 version

File: pom.xml [hunk 2]
```diff
@@ -15,8 +15,8 @@
...
+    <vertx.version>4.3.3</vertx.version>
     <junit.jupiter.version>5.8.2</junit.jupiter.version>
     <rest-assured.version>5.1.1</rest-assured.version>
     <log4j-slf4j-impl.version>2.17.2</log4j-slf4j-impl.version>
```
Comment (grabsefx) [inline pom.xml:22]: please upgrade to the latest 2.19.0 version

Comment (SinghAdes) [reply pom.xml]: fixed

File: pom.xml (hunk 1 above)
Comment (SinghAdes) [reply pom.xml]: fixed

File: pom.xml [hunk 3]
```diff
@@ -15,8 +15,8 @@
 
   <properties>
     <aspectj.version>1.9.7</aspectj.version>
-    <raml-module-builder.version>34.1.0</raml-module-builder.version>
-    <vertx.version>4.3.1</vertx.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version>
+    <vertx.version>4.3.3</vertx.version>
```
[the remark this answers is not in the document]
Comment (grabsefx) [reply pom.xml:19]: @SinghAdes sorry. I meant v4.3.4

[the remark this answers is not in the document]
Comment (SinghAdes) [reply pom.xml]: fixed.

**Review by grabsefx — APPROVED** [review]

**Pull request merged**